### ISSUE TITLE
Auto-detect and switch Language on Quick-Setup.html

### ISSUE THREAD

**Issue by willwade** [AceCentre/pasco#104]
Just wondering if we need to somehow detect the locale and switch to that - e.g. https://stackoverflow.com/a/29106129/1123094

Only on the quick setup screen. 

Also need to figure out how we distribute the iOS app in different regions

**Comment by willwade** [comment]
See also #197

**Comment by gavinhenderson** [comment]
@willwade I have a feeling this now happens?

**Comment by willwade** [comment]
@gavinhenderson not quite. If you are running a non-english lang on iOS it will still load the english locale. Only if you change the locale manually will it auto change. Its really not a priority this - just makes pushing it to other languages and people setting it up easier

**Comment by gavinhenderson** [comment]
@willwade Makes sense, ill keep this issue about then :)
